dev/integration_tests/wide_gamut_test/ios/Runner/DummyPlatformView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  DummyPlatformView.swift
+//  Runner
+//
+//  Created by Jonah Williams on 11/21/23.
+//
+
+import Foundation
+
+import Flutter
+import UIKit
+
+class FLNativeViewFactory: NSObject, FlutterPlatformViewFactory {
+    private var messenger: FlutterBinaryMessenger
+
+    init(messenger: FlutterBinaryMessenger) {
+        self.messenger = messenger
+        super.init()
+    }
+
+    func create(
+        withFrame frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?
+    ) -> FlutterPlatformView {
+        return FLNativeView(
+            frame: frame,
+            viewIdentifier: viewId,
+            arguments: args,
+            binaryMessenger: messenger)
+    }
+
+    public func createArgsCodec() -> FlutterMessageCodec & NSObjectProtocol {
+          return FlutterStandardMessageCodec.sharedInstance()
+    }
+}
+
+class SolidColorView: UIView {
+  private var _started: Bool = false;
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    setupView()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+    setupView()
+  }
+
+  private func setupView() {
+    backgroundColor = .blue
```

Comment:
In order to duplicate the problem, shouldn't the wide gamut color come from the platform view?

Reply:
No, the problem is the overlay flutter view